wsl/how-to/install-ubuntu-wsl2.md

```diff
@@ -4,25 +4,26 @@
 ## What you will learn:
 
 * How to enable and install WSL on Windows 10 and Windows 11
+* How to install `Ubuntu 24.04 LTS` using: Microsoft Store, WSL CLI, or Winget 
 
 ## What you will need:
 
 * A Windows 10 or Windows 11 physical or virtual machine with all the updates installed
 
 ## Install WSL
 
-WSL can be installed from the command line. Open a PowerShell prompt as an Administrator (we recommend using Windows Terminal) and run:
-```
+You can install WSL from the command line. Open a PowerShell prompt as an Administrator (we recommend using [Windows Terminal](https://github.com/microsoft/terminal)) and run:
```

Comment:
I usually prefer to add when in doubt (deleting is cheap ;) Chose to use github because the readme describes installation methods (starting with Store) I guess I'll link directly to the heading https://github.com/microsoft/terminal?tab=readme-ov-file#installing-and-running-windows-terminal to avoid showing github's list of files and suggest we're telling ppl to build the Terminal app before they can start installing WSL, On the other hand, they're on Windows and want to get on the 'Linux-adventure' ;) 
I was also considering https://learn.microsoft.com/en-us/windows/terminal/install but it's last updated in 2022 so again, GH should probably be the most up-to-date single source of truth.